src/components/atoms/ProjectTypeBanner/ProjectTypeBanner.tsx

```diff
@@ -19,56 +19,57 @@ interface ProjectTypeBannerProps {
   projectTypeText: string
 }
 
+export const getImage = (projectType: ProjectType) => {
```

Comment:
Are you sure there was not already something done by @crimson-med to display these images in project's details page?

Reply:
Yes. I reuse this function. Since the pdf package needs is own Type of image, couldn't use this banner component